components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineView.kt

```diff
@@ -192,4 +216,8 @@ class GeckoEngineView @JvmOverloads constructor(
         geckoView.visibility = visibility
         super.setVisibility(visibility)
     }
+
+    companion object {
+        private const val DARK_COVER = 0xFF2A2A2E.toInt()
```

Comment:
nit: should we make this configurable / add a param with a default value?

Reply:
Where should I be adding this param 🤔 ? Or do you just mean make it a mozac color that Fenix can override?

Reply:
Yes, but let's keep that for later. Not sure we need it.